src/App.tsx

```diff
@@ -28,33 +123,22 @@ export const App = () => {
       </form>
 
       <section className="TodoList">
-        <article data-id="1" className="TodoInfo TodoInfo--completed">
-          <h2 className="TodoInfo__title">
-            delectus aut autem
-          </h2>
-
-          <a className="UserInfo" href="mailto:Sincere@april.biz">
-            Leanne Graham
-          </a>
-        </article>
-
-        <article data-id="15" className="TodoInfo TodoInfo--completed">
-          <h2 className="TodoInfo__title">delectus aut autem</h2>
-
-          <a className="UserInfo" href="mailto:Sincere@april.biz">
-            Leanne Graham
-          </a>
-        </article>
-
-        <article data-id="2" className="TodoInfo">
-          <h2 className="TodoInfo__title">
-            quis ut nam facilis et officia qui
-          </h2>
-
-          <a className="UserInfo" href="mailto:Julianne.OConner@kory.org">
-            Patricia Lebsack
-          </a>
-        </article>
+        {toDoList.map(todo => {
+          return (
+            <article
+              data-id={todo.id}
+              className="TodoInfo TodoInfo--completed"
+            >
+              <h2 className="TodoInfo__title">
+                {todo.title}
+              </h2>
+
+              <a className="UserInfo" href="mailto:Sincere@april.biz">
+                {copy.find(user => user.id === todo.userId)?.name}
```

Comment:
don't call the function from the 'template render', move it to state

Reply:
fixed